src/keycode.zig

```diff
@@ -317,10 +316,42 @@ pub const Keycode = struct {
     ///
     /// ## Version
     /// This function is available since SDL 3.2.0.
-    pub fn isScancode(
-        self: Keycode,
-    ) bool {
-        return C.SDLK_SCANCODE_MASK & self.value > 0;
+    pub fn isScancode(self: Keycode) bool {
+        return C.SDLK_SCANCODE_MASK & @intFromEnum(self) != 0;
+    }
+
+    /// Create an unmanaged keycode from a keycode enum.
+    ///
+    /// ## Function Parameters
+    /// * `self`: keycode enum to make unmanaged.
+    ///
+    /// ## Return Value
+    /// Returns an unmanaged SDL keycode.
+    ///
+    /// ## Remarks
+    /// This makes a copy of the keycode provided.
+    ///
+    /// ## Version
+    /// This function is provided by zig-sdl3.
+    pub fn toSdl(self: ?Keycode) C.SDL_Keycode {
+        return if (self) |val| @intFromEnum(val) else C.SDLK_UNKNOWN;
+    }
+
+    /// Create a keycode enum from an SDL keycode.
+    ///
+    /// ## Function Parameters
+    /// * `event`: SDL keycode to manage.
+    ///
+    /// ## Return Value
+    /// A managed keycode enum.
+    ///
+    /// ## Remarks
+    /// This makes a copy of the keycode provided.
+    ///
+    /// ## Version
+    /// This function is provided by zig-sdl3.
+    pub fn fromSdl(key_code: C.SDL_Keycode) Keycode {
```

Comment:
This does not handle the case where `key_code` is `SDLK_UNKNOWN` and should return `null`.